### ISSUE TITLE
"ERROR:  could not identify an equality operator for type json" thrown when attempting to anonymize a postgres table that contains json columns

### ISSUE THREAD

**Issue by snowflakekiller** [jerometwell/pynonymizer#66]
**Describe the bug**
When attempting to run pynonymizer against a postgres table that includes json columns, an error: 

`ERROR:  could not identify an equality operator for type json` 

is thrown.

Even if the user runs a strategy against a non-json column. As long as there is a column in the table of type json, the error will be thrown

**To Reproduce**

run pynonymizer against a postgres table with any json column. Even if you aren't anonymizing that particular json column, the error will be thrown. 

an example strategyfile snippet:

```
tables:
  user:
    columns:
      friendly_name:
        type: fake_update
        fake_type: user_name
      phone_number:
        type: fake_update
        fake_type: phone_number
```

there is an additional column, `lead_score` of type jsonb present in the `user:` table, that is not mentioned in the strategyfile, but still seems to trigger the `could not identify an equality operator for type json` error

**Expected behavior**

I expect that when anonymizing a table, whether or not a column in the table is of type json, I should be able to anonymize other columns 

**Additional context**
ran against a postgres 11.11 DB with the latest version of pynonymizer installed via pip install pynonymizer 

Also - Thank you very much for creating this tool, it has been very useful

**Comment by jerometwell** [comment]
This definitely seems like a bug! I'll try and get a replication case into the postgres integration test (sakila/pagila db). If i understand this correctly, simply adding the json column should be enough to break it.

**Comment by jerometwell** [comment]
https://github.com/jerometwell/pynonymizer/runs/2470290018?check_suite_focus=true

Well, this is entertaining. This happens because the subqueries have a `WHERE table=table` clause to correlate the inner subqueries with the outer query (so it doesn't cache the query result/calculate it once per table). But json types can't be compared like that, so it fails. 

I've got a more robust way of doing this so i'll fix this in the next few days.

**Comment by jerometwell** [comment]
Hi, Thanks for being patient with this one. I've just released [v1.20.0](https://pypi.org/project/pynonymizer/1.20.0) which should contain this fix! 
Can you take a look and close the issue if this solves it for you? 😇

**Comment by snowflakekiller** [comment]
This solved it, thank you very much!
